Sources/Widgets/SVG/SVGLandmarkRepresentation/index.js

```diff
@@ -132,8 +143,9 @@ export function extend(publicAPI, model, initialValues = {}) {
   macro.setGet(publicAPI, model, [
     'circleProps',
     'fontProperties',
-    'name',
+    'strokeFontProperties',
     'textProps',
+    'showCircle',
```

Comment:
how about a visible property in `circleProps` ?

Reply:
I could do it but there will be changes to do in 3 others widgets (Rectangle, Ellipse and Line). Should I do it in the same PR?

Reply:
if you do change it here, you also need to change in other widgets.

ok to do it all in another PR though